assets/multiqc_config.yml

```diff
@@ -112,6 +112,9 @@ sp:
     fn_re: ".*[kraken2|centrifuge].*report.txt"
   quast:
     fn_re: "report.*.tsv"
+  filtlong:
+    num_lines: 20
+    fn_re: ".*_filtlong.log"
```

Comment:
I think I figured out the multiqc-filtlong problem. The default pattern for the multiqc filtlong module only looks through the 5 first lines, but when running filtlong with short reads the log will start with some additional lines, before reaching the content. So I changed the search pattern for filtlong in the multiqc.config to scan through the first 20 lines. Seems to work.

Reply:
Perfect thank yuo :D

Reply:
Heh, @LilyAnderssonLee familiar :sweat_smile: ?

Reply:
For us, `content_re` pattern is even more than 20 lines.